rpcs3/Emu/NP/np_handler.cpp

```diff
@@ -1109,15 +1115,15 @@ namespace np
 								event = (msg->second.msgFeatures & SCE_NP_BASIC_MESSAGE_FEATURES_BOOTABLE) ? SCE_NP_BASIC_EVENT_INCOMING_BOOTABLE_CUSTOM_DATA_MESSAGE : SCE_NP_BASIC_EVENT_INCOMING_CUSTOM_DATA_MESSAGE;
 								break;
 							case SCE_NP_BASIC_MESSAGE_MAIN_TYPE_GENERAL:
-							case SCE_NP_BASIC_MESSAGE_MAIN_TYPE_ADD_FRIEND:
-							case SCE_NP_BASIC_MESSAGE_MAIN_TYPE_URL_ATTACHMENT:
 								event = SCE_NP_BASIC_EVENT_MESSAGE;
+							case SCE_NP_BASIC_MESSAGE_MAIN_TYPE_ADD_FRIEND:
```

Comment:
There's a missing break. Either add it or add fallthrough

Reply:
Isn't fallthrough implied if there is literally no code?

Reply:
Oops nvm, I see what you meant :D

Reply:
it generates a warning on some compilers iirc